lib/practice_exercises.rb

```diff
@@ -1,13 +1,49 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(m)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  (list.length - 1).times do |count|
+    if list[count + 1] == list[count]
+      list[count + 1] = nil
+    end
+  end
+  
+  (list.length - 1).times do |number|
```

Comment:
I like how this shifts `nil` to the right.  However it misses the problem listed above.

I went ahead and added a test for C13 on this.